tests/packages/install-subdir-exclude/meson.build

```diff
@@ -0,0 +1,45 @@
+# SPDX-FileCopyrightText: 2023 The meson-python developers
+#
+# SPDX-License-Identifier: MIT
+
+project('install-subdir-exclude', version: '1.0.0')
+
+py = import('python').find_installation()
+purelib = py.get_install_dir(pure: true)
+
+# We need two foo directories because of a bug in the Meson metadata generation.
+# https://github.com/mesonbuild/meson/issues/11539
+
+# Examples removed from https://mesonbuild.com/Reference-manual_functions.html#install_subdir
+install_subdir(
+  'foo',
+  install_dir: purelib / 'subdir1',
+  strip_directory : false,
+)
+install_subdir(
+  'foo2',
+  install_dir: purelib / 'subdir2',
+  strip_directory : true,
+)
+install_subdir(
+  'foo/bar',
```

Comment:
This reads a bit funny: the `foo` directory was already installed two `install_subdir()` invocations above. I would pick a different source directory for this example. And same thing for the example below.

Reply:
That requires to copy the entire tree again, that's why I didn't do it. I don't think it really matters either way.